webapp/publisher/cve/cve_views.py

```diff
@@ -52,9 +52,11 @@ def has_cves(snap_name):
             status_code,
         )
 
-    snap_has_cves = CveHelper.has_cve_data(snap_name)
-    if snap_has_cves:
-        return flask.jsonify({"success": True})
+    revisions_with_cves = CveHelper.get_revisions_with_cves(snap_name)
+    if len(revisions_with_cves) > 0:
+        return flask.jsonify(
+            {"success": True, "revisions": revisions_with_cves}
```

Comment:
In other APIs we use `data` for the main data in the request, so in this case it would be `"data": { "revisions": ... }`

There are places we don't do that but they are older endpoints from before we started doing this.